knora/dsplib/utils/xml_upload.py

```diff
@@ -189,6 +191,77 @@ def _convert_ark_v0_to_resource_iri(ark: str) -> str:
     return "http://rdfh.ch/" + project_id + "/" + dsp_uuid
 
 
+def _configure_logger(server: str, shortcode: str, default_ontology: str, verbose: bool) -> dict[str, str]:
+    """
+    Creates a directory structure where the logs will be saved at, depending on the OS, the server, the project, and
```

Comment:
Ah, sorry, I changed this comment because later I understood what you mean. Maybe you could rewrite the sentence to make it clear what folder structure is created and what exactly changes with different OS'.